### PR TITLE
feat: limit obs repeat based on business logic

### PR DESCRIPTION
**Pull request by lucyjemutai**
## Requirements

- [ ] This PR has a title that briefly describes the work done including the ticket number. If there is a ticket, make sure your PR title includes a [conventional commit](https://o3-dev.docs.openmrs.org/#/getting_started/contributing?id=your-pr-title-should-indicate-the-type-of-change-it-is) label. See existing PR titles for inspiration.
- [ ] My work conforms to the [OpenMRS 3.0 Styleguide](https://om.rs/styleguide) and [design documentation](https://zeroheight.com/23a080e38/p/880723-introduction).
- [ ] My work includes tests or is validated by existing tests.

### CODE REVIEW

**Comment by samuelmale** [comment]
@lucyjemutai can you address the merge conflict?

File: src/components/repeat/ohri-repeat.component.tsx [hunk 1]
```diff
@@ -210,11 +209,10 @@ export const OHRIRepeat: React.FC<OHRIFormFieldProps> = ({ question, onChange })
   encounterContext.sessionMode != 'view' &&
     nodes.push(
       <div>
-        {showAddButton(question.questionOptions.repeatOptions, counter) && (
+        {showAddButton(question.questionOptions.repeatOptions?.limit, counter) && (
           <Button
             renderIcon={() => <Add size={16} />}
-            className={styles.repeatButton}
-            kind="tertiary"
+            kind="ghost"
```
Comment (samuelmale) [inline src/components/repeat/ohri-repeat.component.tsx:215]: @lucyjemutai do you need the above change? 

```suggestion
            kind="tertiary"
```

Comment (lucyjemutai) [reply src/components/repeat/ohri-repeat.component.tsx:215]: no

Comment (samuelmale) [reply src/components/repeat/ohri-repeat.component.tsx:215]: I just confirmed that we are using the "ghost" kind.

**Review by samuelmale — APPROVED** [review]

**Pull request merged**